src/web/index.html

```diff
@@ -14,6 +14,7 @@
     <script src="./js/ass.min.js" defer></script>
     <script src="./js/sql.min.js" defer></script>
     <script src="./js/exif-js.min.js" defer></script>
+    <script src="./js/he.min.js" defer></script>
```

Comment:
No need for this dependency, HakuNeko can utilize the underlying Browser to do that with 3 lines of code:
https://github.com/manga-download/hakuneko/blob/a361183c2e96075650f8d83f7861bb3c81a96ad3/src/web/mjs/connectors/MangaLife.mjs#L26